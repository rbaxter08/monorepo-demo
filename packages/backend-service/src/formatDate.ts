import { addDays } from "utils";

export function formatDate(date: Date) {
  return addDays(date, 3).toDateString();
}
