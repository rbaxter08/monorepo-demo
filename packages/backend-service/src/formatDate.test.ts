import { formatDate } from "./formatDate";

describe("formatDate", () => {
  it("formatDate() should format", () => {
    const result = formatDate(new Date("2020-04-13T00:00:00.000+08:00"));
    expect(result).toMatchSnapshot();
  });
});
