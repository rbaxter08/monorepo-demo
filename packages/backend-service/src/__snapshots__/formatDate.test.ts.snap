// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatDate formatDate() should format 1`] = `"Wed Apr 15 2020"`;
